src/main/resources/META-INF/persistence.xml

```diff
@@ -4,11 +4,10 @@
     <provider>org.hibernate.jpa.HibernatePersistenceProvider</provider>
     <class>net.dcess.gestrec.entity.Customer</class>
     <class>net.dcess.gestrec.entity.Invoice</class>
-    <class>net.dcess.gestrec.entity.Account</class>
```

Comment:
delete this persistnce.xml change we don't need it, you should change your port to 3306

Reply:
I changed it before, I don't know why it still appears to you like this.